Services/GEV/DecentralTrainings/classes/class.gevDecentralTrainingGUI.php

```diff
@@ -613,6 +623,16 @@ protected function showSettings($a_form = null) {
 								, "GEV_img/ico-head-create-decentral-training.png"
 								);
 
+		$now = @date("Y-m-d");
+		$start_date = $crs_utils->getStartDate();
+		if ($crs_utils->userHasRightOf($this->current_user->getId(), gevSettings::CANCEL_TRAINING) && 
```

Comment:
Bitte die Regel aus ilObjCourse und hier in eine Methode der CourseUtils rausfaktorisieren.